packages/hathor-rpc-handler/src/rpcMethods/getUtxos.ts

```diff
@@ -27,12 +27,20 @@ const getUtxosSchema = z.object({
     maxUtxos: z.number().default(255),
     token: z.string().default('HTR'),
     filterAddress: z.string(),
-    authorities: z.number().nullable().optional(),
-    amountSmallerThan: z.number().min(0).nullable().optional(),
-    amountBiggerThan: z.number().min(0).nullable().optional(),
-    maximumAmount: z.number().nullable().optional(),
+    authorities: z.number().nullish().optional(),
+    amountSmallerThan: z.number().min(0).nullish().optional(),
+    amountBiggerThan: z.number().min(0).nullish().optional(),
+    maximumAmount: z.number().nullish().optional(),
```

Comment:
I think you forgot to remove the optional on these ones.

Reply:
Done